### PR TITLE
Branch 193119953

### PR DESCRIPTION
**Pull request by yifeif**
Manually resolved
tensorflow/contrib/data/python/kernel_tests/sequence_dataset_op_test.py
tensorflow/contrib/data/python/ops/batching.py
tensorflow/contrib/metrics/python/ops/metric_ops.py
tensorflow/contrib/tensorrt/convert/convert_nodes.cc
tensorflow/core/common_runtime/process_util.cc
tensorflow/core/kernels/cwise_op_clip.cc
tensorflow/core/kernels/cwise_op_clip.h
tensorflow/core/kernels/cwise_op_clip_gpu.cu.cc
tensorflow/core/kernels/maxpooling_op.cc
tensorflow/python/framework/dtypes.py
tensorflow/python/framework/tensor_shape_test.py
tensorflow/python/kernel_tests/clip_ops_test.py
tensorflow/python/ops/clip_ops.py
tensorflow/python/util/tf_inspect.py
tensorflow/tools/api/generator/create_python_api.py
tensorflow/tools/docs/parser.py
tensorflow/tools/docs/parser_test.py

### CODE REVIEW

**Comment by yifeif** [comment]
cc: @jsimsa

File: tensorflow/python/kernel_tests/clip_ops_test.py [hunk 1]
```diff
@@ -124,15 +127,14 @@ def testClipByValueBadShape(self):
...
-        ans = clip_ops.clip_by_value(x, 1.0, clip)
-        tf_ans = ans.eval()
+      with self.assertRaises(ValueError):
+        _ = clip_ops.clip_by_value(x, -clip, clip)
+      with self.assertRaises(ValueError):
+        _ = clip_ops.clip_by_value(x, 1.0, clip)
 
   def testClipByValueNonFinite(self):
-    with self.test_session(use_gpu=True):
+    # TODO(b/78016351): Enable test on GPU once the bug is fixed.
```
Comment (qlzh727) [inline tensorflow/python/kernel_tests/clip_ops_test.py]: For TODOs, do we want to hide them from user? The b/ link does not make much sense here.

**Comment by martinwicke** [comment]
We decided against it because it's hard to make a reversible
transformation, and they are not harmful either. In a perfect world,
everybody would make a GitHub issue to link, but it's pretty far down the
list of important things to do.

**Comment by qlzh727** [comment]
Seems that both py presubmit failed on same test case, need some check and probably rollback.

**Comment by yifeif** [comment]
Internal continuous builds were all green, and so are github builds. I wonder if I mess something up during merge.

**Comment by yifeif** [comment]
Tried reproducing the failure locally and I get a segfault. constant_op_test itself didn't change. Maybe one of the underlying ops it use is causing issues? Any suggestion?

**Comment by yifeif** [comment]
Turns out to be issue with merge indeed! The change in dtypes.py from https://github.com/tensorflow/tensorflow/pull/18481/files got override. Will fix.

**Comment by martinwicke** [comment]
Nice debugging!​

**Pull request closed without merging**